src/base/fagcbot.js

```diff
@@ -2,35 +2,37 @@ const { Client, Collection } = require("discord.js")
 const path = require("path")
 const fetch = require("node-fetch")
 const strictUriEncode = require("strict-uri-encode")
+const { FAGCWrapper } = require("fagc-api-wrapper")
 
 class FAGCBot extends Client {
 	constructor(options) {
 		super(options)
-        
+
 		this.config = require("../../config")
 		this.emotes = this.config.emotes
 
 		// setup rate limit
 		this.RateLimit = new Collection()
 
+		this.fagc = new FAGCWrapper()
```

Comment:
The defination of the wrapper requires the api key to be passed, this is either an issue here or in the defination.